source/extensions/access_loggers/grpc/config_utils.cc

```diff
@@ -19,6 +19,22 @@ getGrpcAccessLoggerCacheSingleton(Server::Configuration::CommonFactoryContext& c
             context.threadLocal(), context.localInfo());
       });
 }
+
+void checkGrpcCluster(const envoy::config::core::v3::GrpcService& config,
+                      Envoy::Upstream::ClusterManager::ClusterInfoMaps all_clusters) {
+  if (config.has_google_grpc()) {
+    return;
+  }
+  const std::string& cluster_name = config.envoy_grpc().cluster_name();
+  const auto& it = all_clusters.active_clusters_.find(cluster_name);
+  if (it == all_clusters.active_clusters_.end()) {
+    throw EnvoyException(fmt::format("Unknown gRPC client cluster '{}'", cluster_name));
+  }
+  if (it->second.get().info()->addedViaApi()) {
+    throw EnvoyException(fmt::format("gRPC client cluster '{}' is not static", cluster_name));
+  }
+}
```

Comment:
this is borrowed from https://github.com/envoyproxy/envoy/blob/main/source/common/grpc/async_client_manager_impl.cc#L49-L57

Reply:
I think that this function should be utilized as a common feature on `ClusterManager`. My consideration is that to put `bool ClusterManager::checkGrpcCluster(const std::string& cluster_name)` and not to throw any exception.  So that After `ClusterManager::checkGrpcCluster` returned false with `ENVOY_LOG`, AccessLogger throws exceptions. WDYT?

Reply:
sg, will do that!

Reply:
done!